forch/faucetizer.py

```diff
@@ -14,6 +14,37 @@
 LOGGER = logging.getLogger('faucetizer')
 
 
+class Device:
```

Comment:
NB: Unless there's significant logic here, you can likely reduce overhead (fewer lines of code) overall just using a PB.  Your call.

Reply:
There are a few logics (e.g., is_fulfilled(), etc.) encapsulated in the Device class. I think there are more pertinent to a Device itself. So here I would keep Device class instead of using protobuf?